jupyterlab/labbox_ephys_widgets_jp/sync_extensions_code.py

```diff
@@ -0,0 +1,60 @@
+import os
+import shutil
+
+
+def sync_extensions_code():
```

Comment:
Here's the script that synchronizes the extensions/ folder

Checks if the folders are identical. If so we are good. If not:

If we are in a dev workspace (as evidenced by a file existing), then we raise an exception.
If we are not in a dev workspace, then we remove any existing extensions/ folder and replace it with a new one.